packages/blockchain-wallet-v4-frontend/src/modals/BuySell/KycRequired/template.tsx

```diff
@@ -136,11 +136,6 @@ const Template: React.FC<Props> = (props) => {
               origin: 'BuySell',
               tier: 2
             })
-            if (props.order) {
```

Comment:
Not sure what was the intention here, and in what context it can be needed.

@milan-bc, any ideas?

Reply:
Actually, this condition never made sense since order is always of Remote type (`Remote.NotAsked` by default). And `!!Remote.NotAsked` is `true`.

Reply:
That is where we really failed to capture the business requirements.

Shall we redirect to `CRYPTO_SELECTION` without any condition? Otherwise, if you completes verification, they will be stuck on this screen. What do you think?